src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
@NOYB Again, it's about consistency. config.xml is trusted above everything else and it is the LEAST likely thing to be incorrect. The user put what they wanted in config.xml, we need to respect that. The system protects itself against invalid XML, the odds that config.xml is corrupt in a way that is still valid XML are very low, and corruption there is only likely to happen in the case of hardware failure and CAs are the least of your worries then.

I know you don't want to admit you're incorrect here, but you can't use "config.xml might be wrong" as the basis of any valid argument. It's not going to happen, and if it does, you have much worse problems than CAs.